Sources/SpeziOnboarding/OnboardingFlow/OnboardingViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import SwiftUI
+
+public class OnboardingViewController: ObservableObject {
+    var views: [AnyView]
```

Comment:
It would be good to associate the view with a unique element in the `NavigationPath`. I would suggest that the `OnboardingNavigationPath` introduces is own type that is used to uniquely identify views in the `NavigationPath` that it maintains and that the `OnboardingFlow` uses the `navigationdestination(for:destination:)` to push and pop elements off the navigation Stack: https://developer.apple.com/documentation/swiftui/view/navigationdestination(for:destination:)

This logic would allow you to easily keep track of the state in the NavigationPath while also allowing a developer to add custom views based on more advanced logic to the `NavigationPath` (as we currently do in the Account part of the Onboarding) flow. 

@Supereg is working on simplifying the setup there so this will not be needed in the future and we can just use this mechanism, but until then and for more advanced use cases being able to modify the `NavigationPath` would be a great addition.